src/highlight.js

```diff
@@ -43,11 +43,11 @@ function createElement({ node, style, useInlineStyles, key }) {
 }
 
 function getLineNumberString(lines, startingLineNumber) {
-  return lines.reduce((lineCountString, _, i) => lineCountString + `${i + startingLineNumber}\n`, '');
+  return lines.map((_line, i) => { return <span className="line" key={"line-" + i}>{`${i + startingLineNumber}\n`}</span> })
 }
 
 function LineNumbers({ codeString, style = {float: 'left', paddingRight: '10px'}, startingLineNumber }) {
-  return <code style={style}>{getLineNumberString(codeString.split('\n'), startingLineNumber)}</code>
+  return <code style={style}>{getLineNumberString(codeString.replace(/\n$/, '').split('\n'), startingLineNumber)}</code>
```

Comment:
So we don't get a line number for a trailing newline. See: 05df3b8f5d679bc11f95b4000c9f6f8fcb4f73c0